doc/functions.sgm

```diff
@@ -532,7 +532,31 @@
             </example>
 
           </sect3>
-            
+          <sect3 id="funcs.spoly.isconvex">
+            <title>
+              Spoly is convex
+            </title>
+            <para>
+              Returns true if spoly is convex.
```

Comment:
Change to "Returns true if the specified spherical polygon is convex."

And what does it return if the polygon is NOT convex? Please add that information as well. Thanks, @ggnmstr!